war/src/main/less/modules/draggable-card.less

```diff
@@ -0,0 +1,44 @@
+.repeated-chunk {
+  &__header {
+    display: flex;
+    align-items: center;
+    justify-content: flex-start;
+    font-weight: bold;
+    margin-bottom: 0.75rem;
+
+    .dd-handle {
+      position: relative;
+      width: 22px;
+      height: 22px;
+      overflow: hidden;
```

Comment:
This causes the heading on components to be randomly indented.

Freestyle build -> Add shell step

![image](https://user-images.githubusercontent.com/21194782/142160067-d0ef3b4c-d410-4a41-9756-0ff18c20e39d.png)